### PR TITLE
Add test-related-sandbox dir entry to style guide

### PR DESCRIPTION
**Pull request by superkhau**
@bajtos @davidcheung PTAL

/cc @strongloop/loopback-dev

### CODE REVIEW

File: style-guide.md [hunk 1]
```diff
@@ -223,6 +223,13 @@ done();
 
 # Testing-related
 
+## Sandbox directories
+
+- All test-related sandbox directories should be inside the `test` dir (ie.
+  `./test/sandbox`)
+- Do not run in random directories like `/tmp/sandbox` as you will run into
```
Comment (davidcheung) [inline style-guide.md]: maybe "Do not create" might be more descriptive 
>Do not run in

File: style-guide.md [hunk 2]
```diff
@@ -251,7 +258,7 @@ beforeEach('some description', function namedFunction() {
 
 The first example shows up in test output, but not stack traces. The second and third example shows up in test output and stack traces, but is a bit redundant to type two descriptions (one in the string and a duplicate in the function name)
 
-> Each of the above styles are acceptible and a decision will be made in the future as to which one is preferred. For now, feel free to pick the one that suits you. 
+> Each of the above styles are acceptible and a decision will be made in the future as to which one is preferred. For now, feel free to pick the one that suits you.
```
Comment (davidcheung) [inline style-guide.md]: acceptable

Comment (superkhau) [inline style-guide.md]: My editor auto deleting trailing spaces. ;)

**Comment by davidcheung** [comment]
LGTM

**Comment by superkhau** [comment]
@strongloop/loopback-dev Note, I had to merge this one in to get the link to the proper section.

**Pull request merged**